src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlOutputKey.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+namespace JuliusSweetland.OptiKey.Models
+{
+    public class XmlOutputKey : XmlKey
```

Comment:
What does an XmlOutputKey do?

Reply:
In the keyboard file, there will be a new key type where you will specify the size, position, and type of 'Output': Scratchpad, SuggestionRow, SuggestionColumn.

Reply:
I don't understand I'm afraid. Would you not use some combination of XmlScratchpad, XmlSuggestionRow, or XmlOutput (I think this, or XmlOutputPanel, is a better name than XmlOutputKey - could you please correct in this PR?)